### ISSUE TITLE
Add Interface to Command Line Arguments in Generating Cscope DB

### ISSUE THREAD

**Issue by j5shi** [brookhong/cscope.vim#12]
I need to add `-C` option to cscope command line when generating cscope db to enable case-insensitive search, is there any way to add this option?

**Comment by j5shi** [comment]
Please review my PR, which implemented this request, thanks.

**Comment by brookhong** [comment]
Thanks, but `-C` seems not working on Windows.

**Comment by j5shi** [comment]
Yes it works, I've already created a pull request to fix this issue, please have a check, thanks.

Sent from my Windows Phone
________________________________
发件人: brookhong<mailto:notifications@github.com>
发送时间: ‎2015/‎5/‎16 18:43
收件人: brookhong/cscope.vim<mailto:cscope.vim@noreply.github.com>
抄送: Jia<mailto:j5shi@live.com>
主题: Re: [cscope.vim] Add Interface to Command Line Arguments in Generating Cscope DB (#12)

Thanks, but `-C` seems not working on Windows.

---
